### DISCUSSION TITLE
Quasar dev crash

### DISCUSSION THREAD

**Discussion by Kishiro80** [quasarframework/quasar#12213] [General - Components / Directives / etc]
Upon running quasar dev or quasar build, this error pop up and when i make any changes to code, it wont be updated on webpage. I need to kill the terminal, and re run the code again. 

Below is the error copied from terminal 

> 

> 
> » Dev mode.......... spa
>  » Pkg quasar........ v2.4.13
>  » Pkg @quasar/app... v3.3.0
>  » Transpiled JS..... yes (Babel)
> 
>  App • Opening default browser at http://localhost:8081
> 
> Issues checking in progress...
> 
> <--- Last few GCs --->
> 
> [1556:000001DC0E5F1FD0]    43424 ms: Mark-sweep (reduce) 2045.9 (2051.3) -> 2045.1 (2052.6) MB, 1446.8 / 0.0 ms  (average mu = 0.092, current mu = 0.008) allocation failure scavenge might not succeed
> [1556:000001DC0E5F1FD0]    45822 ms: Mark-sweep (reduce) 2046.2 (2051.6) -> 2045.4 (2052.3) MB, 2381.9 / 0.0 ms  (average mu = 0.039, current mu = 0.007) allocation failure scavenge might not succeed
> 
> 
> <--- JS stacktrace --->
> 
> FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
>  1: 00007FF79B76046F napi_wrap+109311
>  2: 00007FF79B705156 v8::internal::OrderedHashTable<v8::internal::OrderedHashMap,2>::NumberOfElementsOffset+33302
>  3: 00007FF79B705F26 node::OnFatalError+294
>  4: 00007FF79BFD2B4E v8::Isolate::ReportExternalAllocationLimitReached+94
>  5: 00007FF79BFB792D v8::SharedArrayBuffer::Externalize+781
>  6: 00007FF79BE61CCC v8::internal::Heap::EphemeronKeyWriteBarrierFromCode+1516
>  7: 00007FF79BE6D04A v8::internal::Heap::ProtectUnprotectedMemoryChunks+1258
>  8: 00007FF79BE6A1F9 v8::internal::Heap::PageFlagsAreConsistent+2457
>  9: 00007FF79BE5EDE1 v8::internal::Heap::CollectGarbage+2033
> 10: 00007FF79BE5D005 v8::internal::Heap::AllocateExternalBackingStore+1317
> 11: 00007FF79BE7D2A7 v8::internal::Factory::NewFillerObject+183
> 12: 00007FF79BBACC31 v8::internal::interpreter::JumpTableTargetOffsets::iterator::operator=+1409
> 13: 00007FF79C05B50D v8::internal::SetupIsolateDelegate::SetupHeap+463949
> 14: 00000112AD3253C3
> RpcIpcMessagePortClosedError: Process 1556 exited with code "134" [null]
> Issues checking service aborted - probably out of memory. Check the `memoryLimit` option in the ForkTsCheckerWebpackPlugin configuration.
> If increasing the memory doesn't solve the issue, it's most probably a bug in the TypeScript or EsLint.

**Comment by rstoenescu** [comment]
It seems that you ran out of RAM memory. Free up some.

**Reply by Kishiro80** [reply]
Not a RAM issue. Current total usage 6GB, I have 14 GB of useable RAM. where can i check the memory limit?

**Reply by rstoenescu** [reply]
Increase Node's mem limit. "node --max-old-space-size=8192 quasar dev/..."

**Reply by Kishiro80** [reply]
i got error.. but i know this is due to my lack of knowledge..

> Error: Cannot find module 'C:\Users\User\Documents\Develop\EMCOMP\quasar'
>     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
>     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
>     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
>     at internal/main/run_main_module.js:17:47 {
>   code: 'MODULE_NOT_FOUND',
>   requireStack: []
> }
